code/go/0chain.net/core/util/markle_patricia_trie_interface.go

```diff
@@ -91,12 +91,13 @@ func GetPruneStats(ctx context.Context) *PruneStats {
 }
 
 const (
-	PruneStateStart     = "started"
-	PruneStateUpdate    = "updating"
-	PruneStateSynch     = "synching"
-	PruneStateDelete    = "deleting"
-	PruneStateCommplete = "completed"
-	PruneStateAbandoned = "abandoned"
+	PruneStateStart         = "started"
+	PruneStateUpdate        = "updating"
+	PruneStateSynch         = "synching"
+	PruneStateDelete        = "deleting"
+	PruneStateCommplete     = "completed"
+	PruneStateAbandoned     = "abandoned"
+	MPTMaxAllowableNodeSize = 1000000 // 1 MB
```

Comment:
We'd better put this const in another place that separate it from the PruneState, so perhaps put it in the head of the file above the `ErrIteratingChildNodes`.